subiquity/server/controllers/filesystem.py

```diff
@@ -507,22 +505,17 @@ def stop_listening_udev(self):
         loop.remove_reader(self._monitor.fileno())
 
     def _udev_event(self):
-        cp = run_command(['udevadm', 'settle', '-t', '0'])
-        if cp.returncode != 0:
-            log.debug("waiting 0.1 to let udev event queue settle")
-            self.stop_listening_udev()
-            loop = asyncio.get_event_loop()
-            loop.call_later(0.1, self.start_listening_udev)
-            return
```

Comment:
I'm not sure why you're removing this?

Reply:
It made sense when I was turning off the udev listener, but less so now.  Restored.